docusign/1.0/modules/ROOT/pages/docusign-connector-reference.adoc

```diff
@@ -1,8 +1,10 @@
-= Docusign Connector 1.0 Reference
+= Docusign Connector 1.0 Reference - Mule 4
 include::reuse::partial$non-inclusive-banner.adoc[]
 
 Anypoint Connector for DocuSign (DocuSign Connector) enables you to access the DocuSign platform and exposes operations provided by the DocuSign eSignature API v2.1.  Visit the https://developers.docusign.com/docs/esign-rest-api/reference/[DocuSign API reference site] to learn more.
 
+NOTE: The connector does not perform validation on certain fields, such as *Envelope Id* or *Document Id*, so you must validate input values for these fields before sending them to the API and invoking the connector's operations and sources. For example, you can use the xref:validation-connector::validation-examples.adoc#configure-the-matches-regex-operation[*Matches regex* operation from Validation Module] to validate input values for these fields.
```

Comment:
I would add here why they should validate the inputs - that otherwise there is a potential path traversal attack possibility.